scripts/makeetl

```diff
@@ -13,7 +13,8 @@ OP_INPUT_PATH=$(HOOT_HOME)/$(INPUT_PATH)
 
 # This replaces semicolon with vsizip and path
 ifeq "$(INPUT_TYPE)" "ZIP"
-	OP_INPUT="/vsizip/$(HOOT_HOME)/$(INPUT_PATH)/$(subst ;,/" "/vsizip/$(HOOT_HOME)/$(INPUT_PATH)/,$(INPUT))/"
+#	OP_INPUT="/vsizip/$(HOOT_HOME)/$(INPUT_PATH)/$(subst ;,/" "/vsizip/$(HOOT_HOME)/$(INPUT_PATH)/,$(INPUT))/"
+	OP_INPUT="/vsizip/$(HOOT_HOME)/$(INPUT_PATH)/$(subst ;," "/vsizip/$(HOOT_HOME)/$(INPUT_PATH),$(INPUT))"
```

Comment:
Thanks @mattjdnv for finding this.  Zipped up Shapefiles were failing without this little change.